Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -683,7 +760,7 @@ class SpringAnimation extends Animation {
   }
 }
 
-type ValueListenerCallback = (state: {value: number}) => void;
+type AnimatedListenerCallback = (state: Object) => void;
```

Comment:
Since the callbacks are now added by the parent class (AnimatedWithChildren), the arguments are now up to the extending classes, which means AnimatedValueXY takes `{x: number, y:number}` while AnimatedValue takes `{value: number}`.
I'd like to keep the types but I'm not familiar enough with Flow to know how that would work. Should I investigate further?